controller/coq_lsp.ml

```diff
@@ -501,15 +500,15 @@ let rec process_queue ofmt ~state =
     ignore (Queue.pop request_queue);
     let dict = U.to_assoc com in
     let m = string_field "method" dict in
-    LIO.log_info "process_queue" ("Serving Request: " ^ m);
+    LIO.trace "process_queue" ("Serving Request: " ^ m);
     (* We let Coq work normally now *)
     Control.interrupt := false;
     dispatch_message ofmt ~state dict);
   process_queue ofmt ~state
 
 let lsp_cb oc =
   Fleche.Io.CallBack.
-    { log_error = LIO.log_error
+    { log_error = LIO.trace ?extra:None
```

Comment:
Maybe rename this field to `trace_verbose`, and perhaps introduce a `trace_message`?

Reply:
Renamed, I think just `trace` + extra info for verbose is ok.